spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -142,6 +142,9 @@ A full list of changes delivered in the 2.0 release can be found at link:https:/
 
 === Spec Changes
 
+- Property expansion is a MicroProfile Config 2.0 feature. As such, previous versions of the specification would not
```

Comment:
You need to rebase this with the latest release note. You need to list this in Incompatible Changes to clearly spec the Config.getValue change.